src/main/java/ch/njol/skript/expressions/ExprTamer.java

```diff
@@ -52,10 +52,9 @@
 	@Override
 	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
 		if (!ScriptLoader.isCurrentEvent(EntityTameEvent.class)) {
-			Skript.error("tamer expression can be only used in entity tame event.");
+			Skript.error("the `tamer` expression may only be used in entity tame events.");

```

Comment:
There is only one tame event, and you aren't able to use it anywhere else so `may` wouldn't be correct.

Reply:
All Njol's Skript errors say `may` so i'm just making it correct in the grammar.

Reply:
May, as in 'you shouldn't do that, it does not work' is fine.